src/components/DescriptionList/DescriptionListItem.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+import { Heading, HeadingProps } from '../Heading'
+
+export interface DescriptionListItemProps {
+  label: string
+  labelTag?: HeadingProps['tag']
+  children: React.ReactNode
```

Comment:
[MUST] Use `term, description` instead of `label, children` to specify that it is a `DefinitionList`.